docs/developer-guide/local-development-gotchas.md

```diff
@@ -1,7 +1,30 @@
-# Local development gotchas
+## How do I disable Django's 'debug' mode?
 
-This page lists common local development issues and how do resolve them.
+Sometimes you need to disable Django's DEBUG mode in order to test certain behaviours. To do this, you have two options:
 
-## Brief description of problem
+1. Add `DEBUG = False` to `config/settings/local.py` 
+2. Add `DEBUG=False` to your `.env` file
 
-Solution description goes here.
+## How do I disable Django Debug Toolbar?
+
+If you don't need Django's debugging tools, setting `DEBUG=False` as detailed above will disable Django Debug Toolbar also.
+
+If you still need Django's debugging tools, but don't want to use Django Debug Toolbar for any reason, you have two options for disabling it:
+
+1. Add `DEBUG_TOOLBAR_ENABLED = False` to `config/settings/local.py` 
+2. Add `DEBUG_TOOLBAR_ENABLED=False` to your `.env` file
+
+## How do I run with 'DEBUG=False' in dev ?
```

Comment:
Would prefer something non-techy, like:
```
How do I disable Django's 'Debug mode' locally?
```

Reply:
Done. Thank you Andy for these comments docs